src/components/ContentSidebar/ActivityFeed/activity-feed/activityFeedUtils.js

```diff
@@ -27,14 +29,17 @@ export function collapseFeedState(feedState: ?FeedItems): FeedItems {
             previousFeedItem.action === ItemTypes.upload
         ) {
             const {
-                modified_by: prevModifiedBy,
```

Comment:
Not sure if this is worth renaming. This is still the previous modified by user, it just may be null

Reply:
i could not rename, but then that approach I would need to use a variable I can modify, since I need to do the check to fallback onto `PLACEHOLDER_USER` instead of `const`. Any preference?

Reply:
resolved, we agreed this is cool.